packages/core/src/components/popover-canvas/readme.md

```diff
@@ -27,20 +27,51 @@ use the `referenceEl` prop rather than the `selector` the referenced element can
 </TdsPopoverCanvas>
 
 ```
+<br>
+### Controlled vs Uncontrolled
+
+#### Controlled mode
+The popover component will be shown or hidden based on a `trigger` prop and `selector` or `referenceEl` prop.
+
+Example:
+
+```html
+<button id="trigger">Open Popover</button>
+<tds-popover-canvas [selector]="#trigger">
+  <h2 class="tds-headline-02 tds-u-mt0">A Popover Canvas!</h2>
+</tds-popover-canvas>
+```
+
+#### Uncontrolled mode
+The `open` prop can be used to control the visibility of the popover, meaning that the popover will be shown or hidden based on the `open` prop. Props `selector` or `referenceEl` are still needed in order to determine the position of the popover. 
+
+Example:
+
+```html
+<button id="trigger" onClick="{() => showPopover = !showPopover}">Open Popover</button>
+<tds-popover-canvas id="popover" selector="#trigger" open="showPopover">
+  <h2 class="tds-headline-02 tds-u-mt0">A Popover Canvas!</h2>
+</tds-popover-canvas>
+
+```
```

Comment:
I think these should be swapped. When I as a user controlls the open state it is controlled. When I rely on the default logic it is uncontrolled.

Reply:
My bad :(